packages/orchestration/src/orchestration-api.ts

```diff
@@ -86,8 +87,6 @@ export interface Chain<CI extends ChainInfo> {
    */
   makeAccount: () => Promise<OrchestrationAccount<CI>>;
   // FUTURE supply optional port object; also fetch port object
-
-  // TODO provide a way to get the local denom/brand/whatever for this chain
```

Comment:
`getDenomOn()` is intended to provide this.

It's on `Orchestrator` instead of `Chain` because the `Chain` implementation doesn't have ready access to `chainHub`. Perhaps we shouldn't let internal details like that drive the external API, but it was the easiest thing that seems to work.

Reply:
intended design is/was:

```js
osmosis.getLocalDenom(baseDemon)
```